### PR TITLE
Selectable version

### PR DESCRIPTION
**Pull request by aboganas**
Allow the user to select td-agent version

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 ---
 # defaults file for ansible-tdagent
 
+# td-agent version e.g 2 ,3 ..
+td_agent_version: 3
```
Comment (lrk) [inline defaults/main.yml]: does it require handling "major / minor" version number.
If yes, rename to "td_agent_repository_major_version" or use ansible filter if available to extract the major version part.

**Review by lrk — CHANGES_REQUESTED** [review]

File: defaults/main.yml (hunk 1 above)
Comment (aboganas) [reply defaults/main.yml]: this only controls the major version as the os by default would install and update the latest minor version. so If you want I can rename it to "td_agent_repository_major_version"

**Comment by Berfou** [comment]
Hi everyone, sorry to pop up in the PR, but I also need a way to select a major version.
What is missing to continue the review and merge the PR ?
Thanks

**Review by lrk — APPROVED** [review]

**Comment by lrk** [comment]
@Berfou sorry,just missing time ;)

**Comment by Berfou** [comment]
You're welcome!! thanks you very much!

**Pull request merged**